evap/student/templates/student_vote.html

```diff
@@ -14,8 +14,8 @@
 
 {% block content %}
     {{ block.super }}
-    {% if errors %}
-        <div class="alert alert-danger" role="alert">{% blocktrans %}Please make sure to vote for all contributors or click on "I can't give feedback" to skip a person.{% endblocktrans %}</div>
+    {% if is_bound %}
```

Comment:
ok, i now know why it works, but why didn't "if errors" work as well? seems more explicit to me.